proposal.rst

```diff
@@ -152,14 +150,14 @@ Motivations
 
 **Why have you chosen this particular project?**
 
-FILLME
+I have read several articles about how Google search works and it is get interesting to me to have some experience in this area.  
 
 **Who will benefit from your project and in what ways?**
 
 .. For example, think about the likely user-base, what they currently have to
 .. do and how your project will improve things for them.
 
-FILLME
+Firstly, I want to get skills of interacting with people inside project. Secondly, it is very important to try to understand code wich written by other people.  
```

Comment:
The question is more aiming to start you thinking about other people your work might benefit.

Once we have some idea who we might be writing the code for, we can use that to help make decisions about the code.  For example, if you're writing a word processor, thinking about the expected users can help you decide how important it is that the user interface should be translatable into other languages, the relative importance of various features (e.g. people writing for publications find a "word count" feature important, but for many people it's never used), and so on.